devices/Bmm150/packages.config

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="nanoFramework.CoreLibrary" version="1.10.5" targetFramework="netnanoframework10" />
+  <package id="nanoFramework.Hardware.Esp32" version="1.3.3-preview.22" targetFramework="netnanoframework10" />
+  <package id="nanoFramework.Runtime.Events" version="1.9.1-preview.7" targetFramework="netnanoframework10" />
```

Comment:
This one is not needed on what I can see from your code

Reply:
Indeed. Removed.